index.js

```diff
@@ -7,33 +7,16 @@ module.exports = {
   name: require('./package').name,
 
   treeForAddonTestSupport(tree) {
-    // intentionally not calling _super here
-    // so that can have our `import`'s be
-    // import { ... } from 'ember-test-helpers';
-
     let input = debugTree(tree, 'addon-test-support:input');
 
-    let compiler = this.project._incrementalTsCompiler;
-    if (this.isDevelopingAddon() && compiler) {
-      // eslint-disable-next-line node/no-unpublished-require
-      let TypescriptOutput = require('ember-cli-typescript/js/lib/incremental-typescript-compiler/typescript-output-plugin');
-      // eslint-disable-next-line node/no-unpublished-require
-      let MergeTrees = require('broccoli-merge-trees');
-
-      let tsTree = debugTree(
-        new TypescriptOutput(compiler, {
-          'addon-test-support/@ember/test-helpers': '@ember/test-helpers',
-        }),
-        'addon-test-support:ts'
-      );
-
-      input = debugTree(
-        new MergeTrees([input, tsTree]),
-        'addon-test-support:merged'
-      );
+    if (this.isDevelopingAddon()) {
```

Comment:
@rwjblue this simplifies development, quickly removing the need to run a separate "TS build" process.
I wouldn't consider this a final product, but demonstrates the viability  "TS in dev + precompiled on publish approach"

When you have time, we should discuss

Reply:
I'd like to understand this more too. I'm not entirely sure what you mean when you refer to "TS in dev + precompiled on publish approach", because that describes what's done today - we use TS in dev, and precompile for releases.

Reply:
the comment was more based on an intermediate step + discussion with rob. Functionally this appears the same, but no longer requires ember-cli-typescript

Reply:
As yes, I remember this. This was the removal of the `incremental-typescript-compiler` from 1.5.0 -> 2.0.0. Makes sense.